roles/openshift_web_console/tasks/install.yml

```diff
@@ -114,15 +114,24 @@
     src: "{{ mktemp.stdout }}/{{ __console_config_file }}"
   register: updated_console_config
 
-- name: Apply the web console template file
-  shell: >
-    {{ openshift_client_binary }} process -f "{{ mktemp.stdout }}/{{ __console_template_file }}"
-    --param API_SERVER_CONFIG="{{ updated_console_config['content'] | b64decode }}"
-    --param IMAGE="{{ openshift_web_console_image_name }}"
-    --param NODE_SELECTOR={{ openshift_web_console_nodeselector | to_json | quote }}
-    --param REPLICA_COUNT="{{ openshift_web_console_replica_count }}"
-    --config={{ mktemp.stdout }}/admin.kubeconfig
-    | {{ openshift_client_binary }} apply --config={{ mktemp.stdout }}/admin.kubeconfig -f -
```

Comment:
We might be able to remove the step above that copies admin.kubeconfig now

Reply:
Disregard, it's used in start.yml

Reply:
Fixed